clients/react-adapter.rst

```diff
@@ -1,2 +1,78 @@
 React Adapter
 =============
+
+An HTTPlug adapter for the `React Http client`_.
+
+Installation
+------------
+
+To install the React adapter, which will also install React itself (if it was
+not yet included in your project), run:
```

Comment:
WDTY of : "Use composer to install the React adapter. It will also install React as it's a dependency."